packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.Props.ts

```diff
@@ -7,11 +7,17 @@ export interface IChoiceGroupProps extends React.HTMLProps<HTMLElement> {
   options: IChoiceGroupOption[];
 
   /**
-   * A callback for receiving a notification when the choice has been changed.
+   * @deprecated
+   * Deprecated and will be removed by v 1.0. Use 'onChange' instead.
```

Comment:
The issue I foresee is that major bumps should happen frequently, but could hypothetically happen close to each other. We don't have a fixed schedule for major bumps; we simply avoid them but will do them when we need to. With that in mind, saying "removed a version X" is really tough because we want to give adequate time for consumers to remove their use of a deprecated feature.

Maybe a better way to schedule these would be a date of expiration; e.g. 3-6 months. "This property will be supported until March 16, 2017, after which will be removed."

If we do it this way, we can major bump when necessary and not feel blocked on deprecations scheduled for a particular version.